themes/default/content/learn/building-with-pulumi/stack-outputs/index.md

```diff
@@ -162,11 +133,12 @@ value for `frontend_port` set (along with some other values we set in the
 Fundamentals tutorial):
 
 ```bash
-$ cat Pulumi.dev.yaml
+$ cat Pulumi.staging.yaml
+
 config:
   my-first-app:backend_port: "3000"
   my-first-app:database: cart
-  my-first-app:frontend_port: "3001"
+  my-first-app:frontend_port: "3002"
   my-first-app:mongo_host: mongodb://mongo:27017
   my-first-app:mongo_port: "27017"
   my-first-app:node_environment: development
```

Comment:
One thing I noticed while walking through these pages is that when I got to this step, the `pulumi up` failed because of ports already in use. I didn't stop to debug (as it wasn't related to the code in this PR), but I _think_ it might be because the `dev` stack is still up and running, and since the backend and Mongo ports are the same in both stacks, Docker failed to create the container(s). I'll doublecheck this now, but wanted to call it out in case anyone else ran into it.

Reply:
It is; I thought that we had them tear down dev and stand up staging. There's a thread somewhere about the issue; I believe the issue was that we couldn't pass the port number to the frontend app's express build somehow. Possibly something to patch when we try to drop the dataseed container.